[2022-07-29 20:51:46,223] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: nse_daily_data_load.Stg_tgt_merge manual__2022-07-30T00:42:07.349428+00:00 [queued]>
[2022-07-29 20:51:46,229] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: nse_daily_data_load.Stg_tgt_merge manual__2022-07-30T00:42:07.349428+00:00 [queued]>
[2022-07-29 20:51:46,229] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 20:51:46,229] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-29 20:51:46,229] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 20:51:46,264] {taskinstance.py:1377} INFO - Executing <Task(SnowflakeOperator): Stg_tgt_merge> on 2022-07-30 00:42:07.349428+00:00
[2022-07-29 20:51:46,267] {standard_task_runner.py:52} INFO - Started process 351626 to run task
[2022-07-29 20:51:46,272] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'nse_daily_data_load', 'Stg_tgt_merge', 'manual__2022-07-30T00:42:07.349428+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/nse_daily_load_stg.py', '--cfg-path', '/tmp/tmpvyl7i0u3', '--error-file', '/tmp/tmpny1y1a_k']
[2022-07-29 20:51:46,273] {standard_task_runner.py:80} INFO - Job 74: Subtask Stg_tgt_merge
[2022-07-29 20:51:46,332] {task_command.py:369} INFO - Running <TaskInstance: nse_daily_data_load.Stg_tgt_merge manual__2022-07-30T00:42:07.349428+00:00 [running]> on host osboxes
[2022-07-29 20:51:46,425] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=nse_daily_data_load
AIRFLOW_CTX_TASK_ID=Stg_tgt_merge
AIRFLOW_CTX_EXECUTION_DATE=2022-07-30T00:42:07.349428+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-30T00:42:07.349428+00:00
[2022-07-29 20:51:46,426] {snowflake.py:118} INFO - Executing:  MERGE into EDW_DEV.EDW.NSE_STOCK_DAILY_DATA_F f1
USING EDW_DEV.EDW.NSE_STOCK_DAILY_DATA_STream stg1
on f1.SYMBOL = stg1.SYMBOL and f1.TIMESTAMP = stg1.TIMESTAMP 
when not matched and stg1.METADATA$ACTION ='INSERT' and stg1.METADATA$ISUPDATE = FALSE 
then
insert (f1.SYMBOL,f1.SERIES,f1.OPEN,f1.HIGH,f1.LOW,f1.CLOSE,f1.LAST,f1.PREVCLOSE,f1.TOTTRDQTY,f1.TOTTRDVAL,f1.TIMESTAMP,f1.TOTALTRADES,f1.ISIN)  
values (stg1.SYMBOL,stg1.SERIES,stg1.OPEN,stg1.HIGH,stg1.LOW,stg1.CLOSE,stg1.LAST,stg1.PREVCLOSE,stg1.TOTTRDQTY,stg1.TOTTRDVAL,stg1.TIMESTAMP,stg1.TOTALTRADES,stg1.ISIN);
[2022-07-29 20:51:46,431] {base.py:68} INFO - Using connection ID 'snowflake_conn_id' for task execution.
[2022-07-29 20:51:46,437] {connection.py:257} INFO - Snowflake Connector for Python Version: 2.7.8, Python Version: 3.8.10, Platform: Linux-5.4.0-74-generic-x86_64-with-glibc2.29
[2022-07-29 20:51:46,438] {connection.py:876} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-07-29 20:51:46,438] {connection.py:894} INFO - Setting use_openssl_only mode to False
[2022-07-29 20:51:47,640] {cursor.py:710} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-07-29 20:51:47,731] {cursor.py:734} INFO - query execution done
[2022-07-29 20:51:47,732] {snowflake.py:328} INFO - Running statement: MERGE into EDW_DEV.EDW.NSE_STOCK_DAILY_DATA_F f1
USING EDW_DEV.EDW.NSE_STOCK_DAILY_DATA_STream stg1
on f1.SYMBOL = stg1.SYMBOL and f1.TIMESTAMP = stg1.TIMESTAMP 
when not matched and stg1.METADATA$ACTION ='INSERT' and stg1.METADATA$ISUPDATE = FALSE 
then
insert (f1.SYMBOL,f1.SERIES,f1.OPEN,f1.HIGH,f1.LOW,f1.CLOSE,f1.LAST,f1.PREVCLOSE,f1.TOTTRDQTY,f1.TOTTRDVAL,f1.TIMESTAMP,f1.TOTALTRADES,f1.ISIN)  
values (stg1.SYMBOL,stg1.SERIES,stg1.OPEN,stg1.HIGH,stg1.LOW,stg1.CLOSE,stg1.LAST,stg1.PREVCLOSE,stg1.TOTTRDQTY,stg1.TOTTRDVAL,stg1.TIMESTAMP,stg1.TOTALTRADES,stg1.ISIN);, parameters: None
[2022-07-29 20:51:47,732] {cursor.py:710} INFO - query: [MERGE into EDW_DEV.EDW.NSE_STOCK_DAILY_DATA_F f1 USING EDW_DEV.EDW.NSE_STOCK_DAI...]
[2022-07-29 20:51:49,633] {cursor.py:734} INFO - query execution done
[2022-07-29 20:51:49,634] {snowflake.py:338} INFO - Statement execution info - {'number of rows inserted': 2224}
[2022-07-29 20:51:49,634] {snowflake.py:342} INFO - Rows affected: 2224
[2022-07-29 20:51:49,635] {snowflake.py:343} INFO - Snowflake query id: 01a5f113-0501-af03-0000-006e0a832389
[2022-07-29 20:51:49,635] {connection.py:507} INFO - closed
[2022-07-29 20:51:49,717] {connection.py:510} INFO - No async queries seem to be running, deleting session
[2022-07-29 20:51:49,963] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=nse_daily_data_load, task_id=Stg_tgt_merge, execution_date=20220730T004207, start_date=20220730T005146, end_date=20220730T005149
[2022-07-29 20:51:50,170] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 20:51:50,467] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
